.github/workflows/publish.yml

```diff
@@ -40,3 +40,11 @@ jobs:
         with:
           push: true
           tags: ethereumoptimism/tokenlist-bot:latest
+
+      - name: Install dependencies to prepare for release 🔧
+        run: yarn
+
+      - name: Build and publish package on NPM 📦
+        run: yarn release
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
I'll be adding the secret later today, will comment and approve this diff once done.

Reply:
@smartcontracts - I know you were creating an account for that, probably faster if you could just add/update `NPM_TOKEN` secret on this repo